tools/DevDiagnostics/DevHome.DevDiagnostics/Pages/SystemResourceUsagePage.xaml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Page
+    x:Class="DevHome.DevDiagnostics.Pages.SystemResourceUsagePage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:models="using:DevHome.DevDiagnostics.Models"
+    mc:Ignorable="d"
+    NavigationCacheMode="Enabled">
+
+    <Page.Resources>
+        <models:HardwareTreeViewTemplateSelector x:Key="HardwareTreeViewTemplateSelector">
+            <models:HardwareTreeViewTemplateSelector.HardwareTemplate>
+                <DataTemplate x:DataType="models:HardwareNode">
+                    <TreeViewItem AutomationProperties.Name="{x:Bind Name}"
+                        Content="{x:Bind Name}"
+                        ItemsSource="{x:Bind Children}" />
+                </DataTemplate>
+            </models:HardwareTreeViewTemplateSelector.HardwareTemplate>
+            <models:HardwareTreeViewTemplateSelector.SensorGroupTemplate>
+                <DataTemplate x:DataType="models:SensorGroupNode">
+                    <TreeViewItem AutomationProperties.Name="{x:Bind Name}"
+                        Content="{x:Bind Name}"
+                        ItemsSource="{x:Bind Children}" />
+                </DataTemplate>
+            </models:HardwareTreeViewTemplateSelector.SensorGroupTemplate>
+            <models:HardwareTreeViewTemplateSelector.SensorTemplate>
+                <DataTemplate x:DataType="models:SensorNode">
+                    <TreeViewItem AutomationProperties.Name="{x:Bind Name}">
+                        <Grid>
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="200"/>
+                                <ColumnDefinition Width="*"/>
+                            </Grid.ColumnDefinitions>
+                            <TextBlock Text="{x:Bind Name, Mode=OneWay}" />
+                            <TextBlock
+                                Grid.Column="1" FontWeight="Bold"
+                                Text="{x:Bind Value, Mode=OneWay}" />
+                        </Grid>
+                    </TreeViewItem>
+                </DataTemplate>
+            </models:HardwareTreeViewTemplateSelector.SensorTemplate>
+        </models:HardwareTreeViewTemplateSelector>
+    </Page.Resources>
+
+    <Grid Margin="12" >

```

Comment:
If the user sizes the window and expands the tree such that it doesn't all fit in the window, do we get a scrollbar?

Reply:
Yes a vertical scroll bar shows up after the tree is expanded